sample/src/main/java/com/alexvasilkov/gestures/sample/activities/PhotoCropActivity.java

```diff
@@ -72,6 +72,7 @@ private void applyFinderShape(boolean animate) {
         GestureController controller = imageView.getController();
 
         // Setting cropping area
+        controller.getSettings().setRestrictBounds(false);
```

Comment:
Perhaps this line of code shouldn't be merged, but this makes it easy to reproduce.